src/bin/cql-stress-cassandra-stress/settings/command/test_user_profile_yamls/full_profile.yaml

```diff
@@ -16,7 +16,7 @@ table_definition:
 queries:
   ins:
     cql: insert into standard1 (pkey, ckey, c1) values (?, ?, ?)
-    consistencyLevel: local
+    consistencyLevel: local_one
     serialConsistencyLevel: local_serial
```

Comment:
Oops, how did it not cause some tests to fail?

Reply:
Previously, we only tested it parsing it to `String`. Now we test parsing into `scylla::Consistency` type as well.